test/TestAppUtils/TestFrame.cs

```diff
@@ -128,6 +136,22 @@ protected override void OnApplyTemplate()
             _unhandledExceptionReportingTextBox = (TextBox)GetTemplateChild("UnhandledExceptionReportingTextBox");
         }
 
+        private void _innerFrameInLabDimensions_Click(object sender, RoutedEventArgs e)
+        {
+            if(Math.Abs(_pagePresenter.MaxWidth - 1024) < 5)
```

Comment:
>  if(Math.Abs(_pagePresenter.MaxWidth - 1024) < 5) [](start = 10, length = 50)

Why the tolerance?

Reply:
No particular reason, do you want me to use a smaller number than 5 for the tolerance?

Reply:
if there not needed can we just check for equality?

Reply:
I'm a bit hesitant to directly compare them as they are floats at the end of the day, but if you say that it won't really matter here, sure I'll use a direct comparison.

Reply:
That makes sense, especially if this value gets dpi scaled, but i think we should be deliberate about the tolerance we choose, scrollview has an epsilon value that I think would be a good starting spot. @gegao18 do you know if this value gets scaled by the system, if so would you suggest scaling our epsilon?

Reply:
I've updated the check. It is now checking for the default value of the presenter MaxWidth, which works since we are resetting the value. That way, we avoid potential issues with precision and DPI scaling.